html_css/intermediate_css/css_math_functions.md

```diff
@@ -87,18 +87,17 @@ If there are not `150px` available, the image will switch to `100%` of the paren
 
 #### max()
 
-Max works the same way as min, only in reverse. It will select the largest possible value from within the parentheses.
-
-The max function is most useful when the viewing window is either exceptionally large, or the user increases the content size by using the browser’s zoom feature. Specifying the maximum possible size of a given element, for example, will prevent that element from infinitely filling the page as it expands.
+Max works the same way as min, only in reverse. It will select the largest possible value from within the parentheses. You can think of `max()` as ensuring a _minimum_ allowed value for a property.
```

Comment:
I feel like the idea of `max()` ensuring a *minimum* value feels a bit counterintuitive and might cause confusion, can you add just a touch of explanation here for why that's the case?